tests/e2e.rs

```diff
@@ -399,6 +399,83 @@ async fn test_e2e_no_phone() {
 	};
 }
 
+#[test(tokio::test)]
+#[test_log(default_log_filter = "debug")]
+async fn test_e2e_sync_invalid_phone() {
+	let mut ldap = Ldap::new().await;
+	ldap.create_user(
+		"John",
+		"Good Phone",
+		"Johnny1",
+		"good_phone@famedly.de",
+		Some("+12015550123"),
```

Comment:
I think we can simplify this test case by removing the good branch, other tests already assert that we can import phone numbers.

Reply:
The good branch is there so that we test updating the good branch to a bad number. I will change the naming to be clearer.